accepted/future-releases/extension-types/feature-specification.md

```diff
@@ -1137,8 +1131,7 @@ Assume that _DV_ is an extension type declaration named `Name`, and
 this case we say that `V1` is a _superinterface_ of _DV_.
 
 If _DV_ does not include an `<interfaces>` clause then _DV_ has
-`Object?` or `Object` as a direct superinterface, according to the subtype
-relation which was specified earlier.
+`Object?` as a direct superinterface.
```

Comment:
I'd still drop this. 
It will just not have any direct superinterface. And that's OK.

(I'm still not sure what it means, since `Object?` is not an interface.)

Reply:
https://en.wikipedia.org/wiki/Deferent_and_epicycle

Reply:
OK, dropped it.